src/Reflow/Parser.hs

```diff
@@ -59,7 +59,7 @@ codeBlock = do
     s <- codeBlockChar
     c <- codeBlockContents
     e <- codeBlockChar
-    void eol
+    void (trailingWhitespace >> eol)
```

Comment:
I don't think I currently care, but I kind of like that every parser returns a value and that it's up to the called to decide if they want to keep it.

Will update to use `$`.